modules/sso.php

```diff
@@ -986,9 +996,18 @@ public function error_msg_enable_two_step( $message ) {
 	 * @return string
 	 */
 	public function error_msg_email_already_exists( $message ) {
-		$err = __( sprintf( 'You already have an account on this site. Please visit your <a href="%1$s">profile page</a> page to link your account to WordPress.com!', admin_url( 'profile.php' ) ) , 'jetpack' );
+		$error = sprintf(
+			wp_kses(
```

Comment:
@lezama same than my comment above. For more background on this https://codex.wordpress.org/I18n_for_WordPress_Developers#HTML